### PR TITLE
Update changelog for 0.8.0

### PR DESCRIPTION
**Pull request by Lnaden**

### CODE REVIEW

**Comment by Lnaden** [comment]
Please double check that credit is correctly attributed for the NWChem and MOPAC features.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -13,6 +13,33 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
+v0.8.0 / 2019-07-18
+-------------------
+
+New Features
+++++++++++++
+
+- (:pr:`95`, :pr:`96`, :pr:`97`, and :pr:`98`) The NWChem interface from QCDB has been added adapted.
```
Comment (loriab) [inline docs/source/changelog.rst]: choose added _or_ adapted. Add jygrace to attribution, too.

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -13,6 +13,33 @@ Changelog
...
+  Thanks to @vivacebelles for this addition!
+- (:pr:`100`) The MOPAC interface has now been added to QCEngine thanks help to from @godotalgorithm.
+
+Enhancements
+++++++++++++
+
+- (:pr:`94`) The gradient and molecule parsed from a GAMESS calculation output file are now returned in ``parse_output``
+- (:pr:`99`) QCEngine now manages an explicit folder for each Psi4 job to write into and passes the scratch directory
+  via ``-s`` command line. This resolves a key mismatch which could cause an error.
```
Comment (loriab) [inline docs/source/changelog.rst]: bug fix more than enhancement?

**Comment by loriab** [comment]
actually, nwchem, gamess, cfour are not registered by default and can't do a full cycle (write missing of write/compute/parse) within qcng. still getting notated on release notes? if we're doing one-time shout-outs for contribs from new packages, Zach hasn't had his for GAMESS, I don't think.

**Pull request merged**